platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SigSet.java

```diff
@@ -167,7 +167,7 @@ public SigSet copy() {
      */
     @Override
     public int getMinimumSupportedVersion() {
-        return ClassVersion.CLEANUP;
+        return ClassVersion.MIGRATE_TO_SERIALIZABLE;
```

Comment:
Restoring the minimum supported version number is not going to be sufficient.  The code path that is related to `MIGRATE_TO_SERIALIZABLE` was removed.  https://github.com/hashgraph/hedera-services/commit/7176c9559053c4503ecca59422046050431d583d

We would need to roll back the commit in the link and restore the `SigInfo` file that was deleted.  

Is there any way we can automate the process for rendering the states for JRS tests when the serialization format changes for the state?   

When the state moves to using ProtoBuf as the serialization format, we're going to have to re-render all the saved states for all tests at some point.